pkgs/by-name/sy/sysdig-cli-scanner/package.nix

```diff
@@ -2,29 +2,42 @@
   stdenv,
   lib,
   fetchurl,
-  makeWrapper,
+  writeShellScript,
 }:
 let
+  pname = "sysdig-cli-scanner";
   versionMetadata = import ./sysdig-cli-scanner.versions.nix;
   fetchForSystem = versionMetadata.${stdenv.system} or (throw "unsupported system ${stdenv.system}");
+
+  wrapper = writeShellScript "${pname}" ''
```

Comment:
I don't know if the name of the wrapper derivation is really that important, since it's not going to be exposed as a package.
For the record, I was following the same constructs as this package: 

https://github.com/NixOS/nixpkgs/blob/e74612b0448ba9a726bbc1e98a99be6224ba4565/pkgs/games/fallout-ce/build.nix